CHANGELOG.md

```diff
@@ -3,10 +3,31 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v1.0.0 - 2024-04-12
```

Comment:
what the release code name?

Reply:
Do we have to?

Reply:
not here, but it's nice to think about it beforehand